awswrangler/dynamodb/_read.py

```diff
@@ -318,7 +321,12 @@ def _read_items(
             # Last resort use Parallel Scan
             executor = _get_executor(use_threads=use_threads)
             dynamodb_client = _utils.client(service_name="dynamodb", session=boto3_session)
-            total_segments = _utils.ensure_cpu_count(use_threads=use_threads)
+
+            if engine.get() == EngineEnum.RAY:
+                total_segments = ray_args.get("parallelism", 100)
```

Comment:
Shall we use autodetect (-1) by default?

Reply:
Maybe we can use [this](https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/util.py#L180)